### PR TITLE
List repository actions should not check branch existence

### LINKED ISSUE #1727: Bug: list actions by branch after branch delete should return old runs
Listing actions by branch checks if a branch exists when it should not.
The information is kept after the branch is deleted and should be available in the API.

### PR DESCRIPTION
**Pull request by nopcoder**
Fix #1727

Removed the check for branch exist for list actions runs.
To fix the test code was required to pass the same block adapter from our catalog to all of other services used by our controller test code.

### CODE REVIEW

File: pkg/api/controller_test.go [hunk 1]
```diff
@@ -1285,3 +1285,95 @@ func TestController_SetupLakeFSHandler(t *testing.T) {
...
+	defer httpServer.Close()
+	// create repository
+	resp, err := clt.CreateRepositoryWithResponse(ctx, &api.CreateRepositoryParams{}, api.CreateRepositoryJSONRequestBody{
+		DefaultBranch:    api.StringPtr("master"),
+		Name:             "repo9",
+		StorageNamespace: "mem://repo9",
+	})
+	verifyResponseOK(t, resp, err)
+	// upload action for pre-commit
+	actionContent := `---
```
Comment (arielshaqed) [inline pkg/api/controller_test.go:1304]: This is quite hard to read.  Can we e.g. indent?

I tried with a random Python YAML parser and with The YPaste Official YAML cgi-bin, and I think that this works (http://ben-kiki.org/ypaste/data/2571/index.html):
```yaml
---
   a:
     - 2
     - 3
   b: yes
   c: no
```

So you might be able to get by with indenting all of the lines.

File: pkg/api/controller_test.go [hunk 2]
```diff
@@ -1285,3 +1285,95 @@ func TestController_SetupLakeFSHandler(t *testing.T) {
...
+	t.Run("total", func(t *testing.T) {
+		respList, err := clt.ListRepositoryRunsWithResponse(ctx, "repo9", &api.ListRepositoryRunsParams{
+			Amount: api.PaginationAmountPtr(100),
+		})
+		verifyResponseOK(t, respList, err)
+		runsCount := len(respList.JSON200.Results)
+		expectedRunsCount := contentCount + 1 // additional one for the actions commit
+		if runsCount != expectedRunsCount {
+			t.Fatalf("ListRepositoryRuns() got %d results, expected %d", runsCount, expectedRunsCount)
+		}
```
Comment (arielshaqed) [inline pkg/api/controller_test.go:1349]: I think this error message will be easier to read.  Also the code is shorter.
```suggestion
		if runsCount != contentCount + 1 { // additional one for the actions commit
			t.Fatalf("ListRepositoryRuns() got %d results, expected %d+1", runsCount, contentCount+1)
		}
```

**Review by arielshaqed — APPROVED** [review]
Neat!

File: pkg/api/controller_test.go (hunk 1 above)
Comment (nopcoder) [reply pkg/api/controller_test.go:1304]: checking if it is possible or an alternative

Comment (nopcoder) [reply pkg/api/controller_test.go]: cloud not indent so I've extract it into a template variable - let me know if it is more readable now.

**Review by arielshaqed — APPROVED** [review]
Great!

Your solution for indentation looking horrible is much simpler & better than my proposal :-)

**Pull request merged**